aperturedb/Connector.py

```diff
@@ -52,6 +52,12 @@
 MESSAGE_LENGTH_FORMAT = '@I'
 MESSAGE_LENGTH_SIZE = struct.calcsize(MESSAGE_LENGTH_FORMAT)
 
+# Should every host - localhost, and ips be required to pass host check for certificates
+CHECK_ALL_HOSTS = int(os.getenv("ADB_SSL_CHECK_ALL_HOSTS", False ))
```

Comment:
I don't think this will handle strings like "true" and "false" well.
See https://github.com/aperture-data/workflows/blob/1e7a6b0d7fe0c08b2e7c1de7c377fb96bcb05e48/base/docker/scripts/wf_argparse.py#L76